.scaffold/tests/phpunit/fixtures/init/php_script/CLAUDE.md

```
@@ -0,0 +1,101 @@
++```php
++putenv('SCRIPT_QUIET=1');
++putenv('SCRIPT_RUN_SKIP=1');
++require 'force-crystal';
++$output = main(['force-crystal', 'arg1'], 2);
++$this->assertStringContainsString('expected', implode("\n", $output));
++```
++
++
+ ### Namespace Structure
+ 
+ - Source code: `YodasHut\App\`
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 0; Actual: 1

(MD007, ul-indent)